src/main/resources/markdown-macro-config.vm

```diff
@@ -0,0 +1,77 @@
+<html>
+<head>
+    <title>Markdown Macro Configuration</title>
+    <meta name="decorator" content="atl.admin">
+    $webResourceManager.requireResource("com.atlassian.auiplugin:ajs")
+    $webResourceManager.requireResource("com.atlassian.auiplugin:aui-toggle")
+    $webResourceManager.requireResource("com.atlassian.auiplugin:aui-forms")
+    #requireResourcesForContext("markdown-configure-resources")
+
+    <link rel="stylesheet" href="//unpkg.com/@atlaskit/css-reset@latest" />
+    <link rel="stylesheet" href="//unpkg.com/@atlaskit/reduced-ui-pack@latest" />
```

Comment:
We can't rely on loading external resources since this could run in an environment where there is no internet connection. This needs to be replaced with local resources instead.